eng/Version.Details.xml

```diff
@@ -25,54 +21,54 @@
       <Uri>https://github.com/dotnet/razor-compiler</Uri>
       <Sha>4e7d2be7c4b831a46b91d763e798f1917a224067</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Configuration" Version="6.0.0" CoherentParentDependency="Microsoft.AspNetCore.Testing">
+    <Dependency Name="Microsoft.Extensions.Configuration" Version="6.0.0">
```

Comment:
Unless you're going to add a runtime subscription, would be much better to remove these from Version.Details.xml entirely. Then, move the versions in Versions.props to a `Manual` block.